fv3core/fv3core/stencils/a2b_ord4.py

```diff
@@ -489,9 +489,14 @@ def __init__(
 
         self.replace = replace
 
-        self._tmp_qx = utils.make_storage_from_shape(self._idx.max_shape)
-        self._tmp_qy = utils.make_storage_from_shape(self._idx.max_shape)
-        self._tmp_qout_edges = utils.make_storage_from_shape(self._idx.max_shape)
+        def make_storage():
+            return utils.make_storage_from_shape(
+                self._idx.max_shape, backend=self._stencil_config.backend
+            )
```

Comment:
The alternative design on this would be to have a `make_storage` method on the `StencilConfig` and `StencilFactory` classes, which makes use of its backend. @rheacangeo and I settled on this design so that you can still make storages without the larger configuration needs of StencilConfig, but it's possible we could enable both. Then here we could have `stencil_factory.make_storage(self._idx.max_shape)`.

Perhaps something to discuss in a team meeting.

Reply:
That makes sense. I'll put this on the team meeting next week.